monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -355,7 +362,7 @@ class ConfigurePageComponent extends AuthComponent {
           { this.state.lastAction === 'invalid_configuration' ?
             <div className="alert alert-danger">
               <i className="glyphicon glyphicon-exclamation-sign" style={{'marginRight': '5px'}}/>
-              An invalid configuration file was imported and submitted, probably outdated.
+              An invalid configuration file was imported or submitted, probably outdated.
```

Comment:
Lets rephrase to 
`An invalid configuration file was imported or submitted.`
The probably outdated is not useful for the user.